thoth/common/logging.py

```diff
@@ -199,23 +218,28 @@ def init_logging(
     Optionally you can specify prefix of the logging environment variable
     determining logging configuration via env vars (defaults to THOTH_LOG_).
     """
-    # TODO: JSON in deployments?
-    # deployed_to_cluster = bool(int(os.getenv('THOTH_CLUSTER_DEPLOYMENT', '0')))
+    if not os.getenv("OPENSHIFT_BUILD_NAME") or int(os.getenv("THOTH_LOGGING_NO_JSON", 0)):
+        # Running outside the cluster or forced to use c
+        formatter = daiquiri.formatter.ColorFormatter(
+            fmt="%(asctime)s %(process)3d %(color)s%(levelname)-8.8s %(name)s:"
```

Comment:
Spacing between `%(color)s` and `%(levelname)`

Reply:
I think it's intentional. The `%(color)` part represents the escape sequence used for different log levels in terminals.

Reply:
oh okay :)